scripts/set_xva_bridge.sh

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+
+# functions
+# usage
+usage()
+{
+    echo "--------------------------------------------------------------------------------"
+    echo "- Usage: $0 [XVA's_filename] compression[zstd|gzip] <bridge_value>"
+    echo "- First and second options are mandatory."
+    echo "- The third option is optional. if nothing is filled, the value will be xenbr0."
+    echo "--------------------------------------------------------------------------------"
+}
+
+# extract and read ova.xml
+info_ovaxml()
+{
+    TMPFOLD=$(mktemp -d /tmp/xvaXXXX)
+    tar -xvf $nameXVA -C $TMPFOLD ova.xml
+    chmod +r $TMPFOLD/ova.xml
+    MyXmlValue=$(grep -oE "<member><name>bridge</name><value>[^<]*</value></member>" $TMPFOLD/ova.xml)
+    MyLenght=${#MyXmlValue}
+    MyLenghtMinus=$(($MyLenght-17))
+    MyNetworkValue=$(cut -c 1-$MyLenghtMinus <<< $MyXmlValue)
+    BridgeNetwork=$(cut -c 35-${#MyNetworkValue} <<< $MyNetworkValue)
+
+    if [ -d $TMPFOLD ]; then
+        rm -Rf $TMPFOLD
+    fi
+}
+
+# check parameters and prompt the usage if needed
+if [ -z "$1" ]
+then
+    echo "xva's name missing!"
+    usage
+    exit 1
+else
+    if [ -e $1 ]; then
+        nameXVA=$(realpath $1)
```

Comment:
There are at least four different ways to name variables in this script. This should be made consistent.

* myVariable
* MyVariable
* myvariable
* MY_VARIABLE